cmd/cluster/context.go

```diff
@@ -162,9 +164,33 @@ func (o *contextOptions) complete(cmd *cobra.Command, args []string) error {
 		o.organizationID = orgID
 	}
 
+	// Initialize external tooling API tokens
+	o.initJIRAToken()
+
 	return nil
 }
 
+func (o *contextOptions) initJIRAToken() {
+	// The order in which these should be loaded would be:
+	// 1. config file
+	// 2. env var
+	// 3. command flag
+	// The highest number should be the one used. So if you have set a value in the config file but pass a different value in the flag, the flag value should be used.
+	// Viper should have already read in the config file at this point.
+
+	// We specifically _don't_ want to use viper's AutomaticEnv functionality for this because we want to leverage the same env vars that the external tooling uses instead of having to set separate env vars
+
+	jiraTokenEnv := os.Getenv("JIRA_API_TOKEN")
```

Comment:
Wouldn't this be more efficient to reverse the order and return if the highest prio one is found before moving on to load the others?  I know it's a trivial thing, but every little bit helps.

Reply:
Proposal:
```
if o.jiratoken != "" {
	viper.Set(utils.JiraTokenConfigKey, o.jiratoken)
}
else if jiraTokenEnv, isSet := os.LookupEnv("JIRA_API_TOKEN"); isSet {
	viper.Set(utils.JiraTokenConfigKey, jiraTokenEnv)
}
```